common/views/components/FeaturedWorkLink/index.tsx

```diff
@@ -34,16 +69,60 @@ const FeaturedWorkLink = ({
   link?: string;
   children?: ReactNode;
 } & HTMLAttributes<HTMLAnchorElement>) => {
+  const portalRef = useRef<HTMLSpanElement>(null);
+  const [portalRight, setPortalRight] = useState(false);
+
+  useEffect(() => {
+    // After the cards are portaled in to the data-portal-id element
```

Comment:
I tried to use PopperJS to deal with keeping the card in the right location, but I couldn't make it work consistently, so opted for a (simpler) `MutationObserver` and resize handler